modules/cnv-updating-access-mode-to-enable-live-migration.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * cnv/live_migration/cnv-live-migration.adoc
+
+[id="cnv-updating-access-mode-to-enable-live-migration_{context}"]
+= Updating access mode to enable LiveMigration
+
+For LiveMigration to function properly, your storage class must use the
+RWX access mode.
+
+Storage class defaults in {CNVProductName} are configured to provide
+ReadWriteOnce (RWO) access mode by default. To enable LiveMigration, the
```

Comment:
Again, I don't think 'enable' is an appropriate word here. LiveMigration requires RWX for the storage class, but it doesn't 'enable' it. You can specify multiple storage classes, with differing settings, in the ConfigMap.

Reply:
I struck this paragraph entirely (see previous comment).

I am striking the word enable, so this module is being renamed _cnv-updating-access-mode-for-live-migration_.